java/pom.xml

```diff
@@ -45,5 +44,6 @@ limitations under the License.
     <module>simple-performance-test</module>
     <module>dataflow-bulk-delete</module>
     <module>quickstart</module>
+    <module>connecting-to-bigtable</module>
```

Comment:
`connect-bigtable` might be better. "ing" not preferred in the package name.

Reply:
Should we do a full quickstart (that connects and reads) that also includes connecting variations?

Reply:
There is already a quickstart that connects and reads. What are you envisioning if that doesn't capture what you want?

https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/blob/master/java/quickstart/src/main/java/com/example/cloud/bigtable/quickstart/Quickstart.java